data/beaconrestapi/src/integration-test/java/tech/pegasys/teku/beaconrestapi/v1/beacon/GetFinalizedCheckpointStateIntegrationTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.beaconrestapi.v1.beacon;
+
+import static javax.servlet.http.HttpServletResponse.SC_OK;
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.pegasys.teku.infrastructure.http.RestApiConstants.HEADER_CONSENSUS_VERSION;
+
+import java.io.IOException;
+import java.util.function.Function;
+import okhttp3.Response;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.teku.api.schema.Version;
+import tech.pegasys.teku.beaconrestapi.AbstractDataBackedRestAPIIntegrationTest;
+import tech.pegasys.teku.beaconrestapi.handlers.v1.beacon.GetFinalizedCheckpointState;
+import tech.pegasys.teku.infrastructure.json.JsonUtil;
+import tech.pegasys.teku.infrastructure.json.types.DeserializableTypeDefinition;
+import tech.pegasys.teku.infrastructure.ssz.SszData;
+import tech.pegasys.teku.infrastructure.ssz.schema.SszSchema;
+import tech.pegasys.teku.spec.SpecMilestone;
+import tech.pegasys.teku.spec.datastructures.state.beaconstate.BeaconState;
+import tech.pegasys.teku.spec.datastructures.state.beaconstate.versions.altair.BeaconStateAltair;
+
+public class GetFinalizedCheckpointStateIntegrationTest
+    extends AbstractDataBackedRestAPIIntegrationTest {
+
+  @Test
+  public void shouldGetBellatrixStateAsJson() throws IOException {
```

Comment:
hmm, the spec currently says this API can return JSON but I'm actually keen to see it only ever return SSZ as the JSON serialisation is expensive and supporting multiple formats makes it much harder to implement this API with a static web server (e.g have a cron job that periodically grabs a state from a real node and just saves it for a static server to serve).

So I think for our implementation we should just go with SSZ only since that's where we'd like to wind up.  If we support JSON now it may be very hard to get rid of later (and potentially causes confusion that with some versions of teku you get json and other ssz by default).

Reply:
I see your point.
Removed it